From 14de72b0aa56e20f24f9ecc1a449da1ed4fd5cda Mon Sep 17 00:00:00 2001
From: Armando Uribe <x0095078@ti.com>
Date: Thu, 22 Jul 2010 20:25:34 -0500
Subject: [PATCH 068/116] staging: tidspbridge: Remove unused macros

This patch removes a couple of macros that are not
being used

Signed-off-by: Armando Uribe <x0095078@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 .../staging/tidspbridge/dynload/dload_internal.h   |    4 ----
 drivers/staging/tidspbridge/dynload/getsection.c   |    1 -
 .../tidspbridge/include/dspbridge/chnldefs.h       |    1 -
 .../tidspbridge/include/dspbridge/chnlpriv.h       |    3 ---
 .../staging/tidspbridge/include/dspbridge/dbdefs.h |   11 -----------
 .../staging/tidspbridge/include/dspbridge/io_sm.h  |    3 ---
 .../staging/tidspbridge/include/dspbridge/mbx_sh.h |   14 --------------
 drivers/staging/tidspbridge/rmgr/disp.c            |    2 --
 drivers/staging/tidspbridge/rmgr/nldr.c            |   13 -------------
 9 files changed, 0 insertions(+), 52 deletions(-)

diff --git a/drivers/staging/tidspbridge/dynload/dload_internal.h b/drivers/staging/tidspbridge/dynload/dload_internal.h
index 5a17e6c..302a7c5 100644
--- a/drivers/staging/tidspbridge/dynload/dload_internal.h
+++ b/drivers/staging/tidspbridge/dynload/dload_internal.h
@@ -310,10 +310,6 @@ extern uint32_t dload_reverse_checksum16(void *data, unsigned siz);
 #endif
 #endif
 
-#define IS_DATA_SCN(zzz) (DLOAD_SECTION_TYPE((zzz)->type) != DLOAD_TEXT)
-#define IS_DATA_SCN_NUM(zzz) \
-		(DLOAD_SECT_TYPE(&dlthis->sect_hdrs[(zzz)-1]) != DLOAD_TEXT)
-
 /*
  * exported by reloc.c
  */
diff --git a/drivers/staging/tidspbridge/dynload/getsection.c b/drivers/staging/tidspbridge/dynload/getsection.c
index 51a7167..e0b3771 100644
--- a/drivers/staging/tidspbridge/dynload/getsection.c
+++ b/drivers/staging/tidspbridge/dynload/getsection.c
@@ -252,7 +252,6 @@ int dload_get_section_info(void *minfo, const char *section_name,
 }
 
 #define IPH_SIZE (sizeof(struct image_packet_t) - sizeof(u32))
-#define REVERSE_REORDER_MAP(rawmap) ((rawmap) ^ 0x3030303)
 
 /**************************************************************************
  * Procedure dload_get_section
diff --git a/drivers/staging/tidspbridge/include/dspbridge/chnldefs.h b/drivers/staging/tidspbridge/include/dspbridge/chnldefs.h
index 0fe3824..5bf5f6b 100644
--- a/drivers/staging/tidspbridge/include/dspbridge/chnldefs.h
+++ b/drivers/staging/tidspbridge/include/dspbridge/chnldefs.h
@@ -40,7 +40,6 @@
 #define CHNL_IOCSTATEOS      0x8000	/* End Of Stream reached. */
 
 /* Macros for checking I/O Completion status: */
-#define CHNL_IS_EOS(ioc)         (ioc.status & CHNL_IOCSTATEOS)
 #define CHNL_IS_IO_COMPLETE(ioc)  (!(ioc.status & ~CHNL_IOCSTATEOS))
 #define CHNL_IS_IO_CANCELLED(ioc) (ioc.status & CHNL_IOCSTATCANCEL)
 #define CHNL_IS_TIMED_OUT(ioc)    (ioc.status & CHNL_IOCSTATTIMEOUT)
diff --git a/drivers/staging/tidspbridge/include/dspbridge/chnlpriv.h b/drivers/staging/tidspbridge/include/dspbridge/chnlpriv.h
index fce5ebd..9292100 100644
--- a/drivers/staging/tidspbridge/include/dspbridge/chnlpriv.h
+++ b/drivers/staging/tidspbridge/include/dspbridge/chnlpriv.h
@@ -50,9 +50,6 @@
 #define CHNL_STATECANCEL	1	/* I/O was cancelled. */
 #define CHNL_STATEEOS		2	/* End Of Stream reached. */
 
-/* Determine if user supplied an event for this channel: */
-#define CHNL_IS_USER_EVENT(mode)  (mode & CHNL_MODEUSEREVENT)
-
 /* Macros for checking mode: */
 #define CHNL_IS_INPUT(mode)      (mode & CHNL_MODEFROMDSP)
 #define CHNL_IS_OUTPUT(mode)     (!CHNL_IS_INPUT(mode))
diff --git a/drivers/staging/tidspbridge/include/dspbridge/dbdefs.h b/drivers/staging/tidspbridge/include/dspbridge/dbdefs.h
index ffcc326..397e2cb 100644
--- a/drivers/staging/tidspbridge/include/dspbridge/dbdefs.h
+++ b/drivers/staging/tidspbridge/include/dspbridge/dbdefs.h
@@ -120,17 +120,6 @@ static inline bool is_valid_proc_event(u32 x)
 	return (x == 0 || (x & VALID_PROC_EVENT && !(x & ~VALID_PROC_EVENT)));
 }
 
-#define IS_VALID_NODE_EVENT(x)    (((x) == 0) || \
-	(((x) & (DSP_NODESTATECHANGE | DSP_NODEMESSAGEREADY)) && \
-	!((x) & ~(DSP_NODESTATECHANGE | DSP_NODEMESSAGEREADY))))
-
-#define IS_VALID_STRM_EVENT(x)     (((x) == 0) || (((x) & (DSP_STREAMDONE | \
-				DSP_STREAMIOCOMPLETION)) && \
-				!((x) & ~(DSP_STREAMDONE | \
-				DSP_STREAMIOCOMPLETION))))
-
-#define IS_VALID_NOTIFY_MASK(x)   ((x) & DSP_SIGNALEVENT)
-
 /* The Node UUID structure */
 struct dsp_uuid {
 	u32 ul_data1;
diff --git a/drivers/staging/tidspbridge/include/dspbridge/io_sm.h b/drivers/staging/tidspbridge/include/dspbridge/io_sm.h
index 841ff94..18aec55 100644
--- a/drivers/staging/tidspbridge/include/dspbridge/io_sm.h
+++ b/drivers/staging/tidspbridge/include/dspbridge/io_sm.h
@@ -30,9 +30,6 @@
 #define IO_SERVICE          2
 #define IO_MAXSERVICE       IO_SERVICE
 
-#define DSP_FIELD_ADDR(type, field, base, wordsize) \
-    ((((s32)&(((type *)0)->field)) / wordsize) + (u32)base)
-
 #ifdef CONFIG_TIDSPBRIDGE_DVFS
 /* The maximum number of OPPs that are supported */
 extern s32 dsp_max_opps;
diff --git a/drivers/staging/tidspbridge/include/dspbridge/mbx_sh.h b/drivers/staging/tidspbridge/include/dspbridge/mbx_sh.h
index 289f6f3..5d165cd 100644
--- a/drivers/staging/tidspbridge/include/dspbridge/mbx_sh.h
+++ b/drivers/staging/tidspbridge/include/dspbridge/mbx_sh.h
@@ -177,22 +177,8 @@
 #define MBX_PM_OPP2 1
 #define MBX_PM_OPP3 2
 #define MBX_PM_OPP4 3
-#define MBX_OLDOPP_EXTRACT(OPPMSG) ((0x00F0 & (OPPMSG)) >> 4)
-#define MBX_NEWOPP_EXTRACT(OPPMSG) (0x000F & (OPPMSG))
-#define MBX_PREVOPP_EXTRACT(OPPMSG) ((0x00F0 & (OPPMSG)) >> 4)
-#define MBX_CUROPP_EXTRACT(OPPMSG) (0x000F & (OPPMSG))
 
 /* Bridge Debug Commands */
 #define MBX_DBG_SYSPRINTF       (MBX_DBG_CLASS + 0x0)
 
-/*
- * Useful macros
- */
-/* DSP-DMA channel */
-#define MBX_SETDDMAVAL(x, y)  (MBX_DDMA_CLASS | (x << MBX_DDMA_BUFSHIFT) | \
-	(y << MBX_DDMA_CHNLSHIFT))
-
-/* Zero-Copy channel */
-#define MBX_SETZCPYVAL(x)  (MBX_ZCPY_CLASS | (x << MBX_ZCPY_CHNLSHIFT))
-
 #endif /* _MBX_SH_H */
diff --git a/drivers/staging/tidspbridge/rmgr/disp.c b/drivers/staging/tidspbridge/rmgr/disp.c
index 29cf6cd..417faeb 100644
--- a/drivers/staging/tidspbridge/rmgr/disp.c
+++ b/drivers/staging/tidspbridge/rmgr/disp.c
@@ -54,8 +54,6 @@
 
 #define CHNLIOREQS      1
 
-#define SWAP_WORD(x)     (((u32)(x) >> 16) | ((u32)(x) << 16))
-
 /*
  *  ======== disp_object ========
  */
diff --git a/drivers/staging/tidspbridge/rmgr/nldr.c b/drivers/staging/tidspbridge/rmgr/nldr.c
index 33d2872..23b44cf 100644
--- a/drivers/staging/tidspbridge/rmgr/nldr.c
+++ b/drivers/staging/tidspbridge/rmgr/nldr.c
@@ -114,19 +114,6 @@
 #define DELETECODEFLAGBIT   5
 #define MAXFLAGS	    6
 
-#define IS_INTERNAL(nldr_obj, segid) (((segid) <= MAXSEGID && \
-	    nldr_obj->seg_table[(segid)] & DYNM_INTERNAL) || \
-	    (segid) == MEMINTERNALID)
-
-#define IS_EXTERNAL(nldr_obj, segid) (((segid) <= MAXSEGID && \
-	    nldr_obj->seg_table[(segid)] & DYNM_EXTERNAL) || \
-	    (segid) == MEMEXTERNALID)
-
-#define SWAPLONG(x) ((((x) << 24) & 0xFF000000) | (((x) << 8) & 0xFF0000L) | \
-	(((x) >> 8) & 0xFF00L) | (((x) >> 24) & 0xFF))
-
-#define SWAPWORD(x) ((((x) << 8) & 0xFF00) | (((x) >> 8) & 0xFF))
-
     /*
      *  These names may be embedded in overlay sections to identify which
      *  node phase the section should be overlayed.
-- 
1.7.1

